### DISCUSSION TITLE
Drives view - add destinetion drop down

### DISCUSSION THREAD

**Discussion by kbyszewski** [teslamate-org/teslamate#2342] [Ideas]
Hi

In a "Drives" view we have "Enter min distance here" and "Enter min speed here"
Can we add a drop down with alphabetic order of all destinations. This way we could quickly select all the drives we have done to particular destination and compare stats. Usually we go to same destination from the same starting point but we could also add a drop down of starting location as well.  

Thanks

**Comment by DrMichael** [comment]
It might be an option to include text filter destinations?

Like in https://github.com/adriankumpf/teslamate/pull/2190

**Comment by DrMichael** [comment]
Well, with Grafana 8 one can add column filters. But, when I add it to Start or Destination, columns Speed and Max Power get filters?!? Apparently not yet bug free.
